spec/requests/app_spec.rb

```diff
@@ -21,4 +21,15 @@
       expect(parsed_response).to eq(data_object_for(verse))
     end
   end
+
+  describe 'GET /devotionals' do
+    let!(:devotionals) { create_list(:devotional, 6) }
+
+    it 'returns a list of devotionals' do
+      get '/devotionals', {}, headers
+
+      expect(last_response.status).to eq(200)
+      expect(parsed_response).to eq(data_object_for(devotionals))
```

Comment:
Since our `devotionals` here in this test is a list of 6 Devotionals, `data_object_for` should catch any extra or missing elements, so I'd say we're covered as far as that `count` vs `size` thing is concerned. 👍